ovirt_imageio/client/_options.py

```diff
@@ -116,8 +116,7 @@ class Parser:
             name="log_file",
             args=["--log-file"],
             config=True,
-            default="/dev/stderr",
-            help="Log file name (default: /dev/stderr).",
+            help="Log file name (default: stderr).",
```

Comment:
This can be confusing for users, they may think that --log-level stderr can work.

Maybe we really need to use:

    default="stderr"

And change the way we configure logging to use stream= instead of filename=.
If we find the special value "stderr" we can use:

    stream=sys.stderr

and otherwise we can create StreamHandler(args.log_file).